scheduler/hooks/retrieve_after_store.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Usage: bash retrieve_after_store.sh <UUID>
+# expected installed env binaries: mktemp, curl, jq
+# env vars used:
+# DEALBOT_CONTROLLER_ENDPOINT
+
+curl -o /tmp/dealbot.$1 $DEALBOT_CONTROLLER_ENDPOINT/tasks/$1\?parsed
+
+if [ "$(cat /tmp/dealbot.$1 | jq '.StorageTask == null')" = "true" ]; then
+  echo "task not a storage task."
+  exit 0
+fi
+
+if [ "$(cat /tmp/dealbot.$1 | jq '.TimeToLastByteMS == null')" = "true" ]; then
+  echo "task did not finish transfer."
+  exit 0
+fi
+
+MINER=$(cat /tmp/dealbot.$1 | jq .StorageTask.Miner -c)
+TAG=$(cat /tmp/dealbot.$1 | jq .StorageTask.Tag -c)
+PAYLOAD=$(cat /tmp/dealbot.$1 | jq .PayloadCID -c)
+
+echo "{\"Miner\": $MINER, \"Tag\": $TAG, \"PayloadCID\": \"$PAYLOAD\", \"CarExport\": false, \"Schedule\": \"$(date -v+5M +"%M %H * * *")\", \"ScheduleLimit\": 1}" | jq -c . > /tmp/dealbot.$1.json
+curl -X POST -d "@/tmp/dealbot.$1.json" -H "Content-Type: application/json" $DEALBOT_CONTROLLER_ENDPOINT/tasks/retrieval
```

Comment:
I think this would be more maintainable with a heredoc, like:

```bash
curl -X POST -d - -H "Content-Type: application/json" $DEALBOT_CONTROLLER_ENDPOINT/tasks/retrieval <<EOF
{
  "Miner": $MINER,
  "Tag": $TAG,
  "PayloadCID": "$PAYLOAD",
  "CarExport": false,
  "Schedule": "$(date -v+5M +"%M %H * * *")",
  "ScheduleLimit": 1
}
EOF
```

* This kind of EOF does expansion like double quotes, so you can still do what you need
* Since the start/end tokens are the EOF word itself, you don't need to quote double quote characters, which is nice for JSON
* Lends itself much better to large strings, since you can split it into multiple lines

Note that I removed the `jq -c` here; as far as I can tell that's a no-op, as long as your expanded JSON here is valid. If it were invalid, I imagine there's no big difference between `jq` or `curl` erroring with a similar message.